index.js

```diff
@@ -130,7 +132,14 @@ const ORDERED_DEFAULTS = {
     commonjs: commonjs({sourceMap: false}),
     json: json(),
     multiEntry: multiEntry({exports: false}),
-    resolve: resolve({browser: true, mainFields: ['module', 'jsnext', 'main']}),
+    resolve: resolve({
+      mainFields: ['browser', 'module', 'jsnext:main', 'main'],
+      dedupe(id) {
```

Comment:
Since modules should have every external, we should dedupe dependencies via that list.